ilc_scrape.py

```diff
@@ -11,6 +11,7 @@
 import requests
 from downloader import DirServer, download_stream
 from utils import print_quit, read_json, sanitize_filepath, sp_args, store_json
+from pathvalidate import sanitize_filename
```

Comment:
The package needs to be added to dependency file using poetry

Reply:
Added package to poetry dependency file. 
Bumped the version as well, is that needed ?